openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -85,59 +85,26 @@
     LIBRARY_BLOCK_DELETED,
     LIBRARY_BLOCK_UPDATED,
 )
+from openedx_learning.core.publishing import api as publishing_api
+from openedx_learning.core.contents import api as contents_api
+from openedx_learning.core.contents.models import MediaType
```

Comment:
I'll make this change, and I agree on putting boundaries between APIs and other apps' models, but I'm not clear on how making this change will help. We're still passing around MediaType model instances.

Reply:
Yeah, it's more to be in the habit of only importing from `api.py`, and if we are using a model directly, it should be one exposed via `api.py`. Ideally in the future we could refactor it to use an [OEP-49 data class](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0049-django-app-patterns.html#data-py) instead of a model.

Reply:
Makes sense, changed. If we want to switch Learning Core to exposing data classes instead of models, we should probably do that sooner than later.